### PR TITLE
Add docs for ALTER RANGE ... RELOCATE

### PR DESCRIPTION
**Pull request by rmloveland**
Fixes DOC-3344

Summary of changes:

- Update the existing 'ALTER RANGE' page to make it a list of
  subcommands (similar to 'ALTER DATABASE', etc.)

- Add a page, 'ALTER RANGE ... RELOCATE', with some info and examples
  about this new command.

- Add some words and back and forth linkage between 'ALTER RANGE
  ... RELOCATE' docs and the existing 'ALTER TABLE
  ... EXPERIMENTAL_RELOCATE' statement docs

### CODE REVIEW

File: v22.1/alter-range-relocate.md [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+
+To alter a range and move a lease or replica between stores, the user must have one of the following:
+
+- Membership to the [`admin`](security-reference/authorization.html#admin-role) role for the cluster.
+
+## Examples
+
+### Find the cluster node IDs and store IDs
+
+To use `ALTER RANGE ... RELOCATE`, you will need to know your cluster's node IDs and store IDs. To get the node IDs and store IDs in your cluster, run the following statement:
```
Comment (lunevalex) [inline v22.1/alter-range-relocate.md:32]: The commands work purely on store id's so you dont need node id's for anything.

File: v22.1/alter-range-relocate.md [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+        46 | /Table/106/1/"seattle"/"ffffffH\x00\x80\x00\x00\x00\x00\x00\x00\x14"                         | ok
+        49 | /Table/106/1/"washington dc"/"L\xcc\xcc\xcc\xcc\xccL\x00\x80\x00\x00\x00\x00\x00\x00\x0f"    | ok
+(9 rows)
+~~~
+
+When it isn't possible to move a lease for a range to the specified store, the `result` column will show the message `unable to find store ...` as shown above.
+
+### Move a replica from one store to another store
+
+If you know the store where a range is located, you can move its lease to another store:
```
Comment (lunevalex) [inline v22.1/alter-range-relocate.md:126]: you can move the replica not the lease.

File: v22.1/alter-range-relocate.md [hunk 3]
```diff
@@ -0,0 +1,228 @@
+---
+title: ALTER RANGE ... RELOCATE
+summary: Use the ALTER RANGE ... RELOCATE statement to move a lease or replica between stores in an emergency situation.
+toc: true
+docs_area: reference.sql 
+---
+
+<span class="version-tag">New in v22.1:</span> The `ALTER RANGE ... RELOCATE` statement is a subcommand of [`ALTER RANGE`](alter-range.html). It is used to move a lease or [replica](architecture/overview.html#architecture-replica) between [stores](cockroach-start.html#store). This is helpful in an emergency situation to relocate data in the cluster.
```
Comment (lunevalex) [inline v22.1/alter-range-relocate.md:8]: I wonder if we should link this to the similar ALTER TABLE EXPERIMENTAL_RELOCATE commands, which are a key based approach to moving replicas and leases around.

File: v22.1/alter-range-relocate.md (hunk 2 above)
Comment (rmloveland) [reply v22.1/alter-range-relocate.md]: Fixed by updating to refer to the replica

File: v22.1/alter-range-relocate.md (hunk 1 above)
Comment (rmloveland) [reply v22.1/alter-range-relocate.md]: Fixed by removing mention of node IDs

**Comment by rmloveland** [comment]
Hi @lunevalex! thanks for looking this over 

does it look good with the latest changes?

File: v22.1/alter-range-relocate.md [hunk 4]
```diff
@@ -0,0 +1,232 @@
...
+       8
+       9
+(9 rows)
+~~~
+
+### Find range ID and leaseholder information
+
+To use `ALTER RANGE ... RELOCATE`, you need to know how to find the range ID, leaseholder, and other information for a [table](show-ranges.html#show-ranges-for-a-table-primary-index), [index](show-ranges.html#show-ranges-for-an-index), or [database](show-ranges.html#show-ranges-for-a-database). You can find this information using the [`SHOW RANGES`](show-ranges.html) statement.
+
+For example, to get all range IDs, leaseholder node IDs, and leaseholder localities for the [`movr.users`](movr.html) table, use the following query:
```
Comment (lunevalex) [inline v22.1/alter-range-relocate.md:62]: change node ID to store ID

File: v22.1/experimental-features.md [hunk 5]
```diff
@@ -55,6 +55,10 @@ To relocate just the lease without moving the replicas, run a statement like the
 > ALTER TABLE t EXPERIMENTAL_RELOCATE LEASE SELECT 1, 'foo';
 ~~~
 
+{{site.data.alerts.callout_info}}
+<span class="version-tag">New in v22.1:</span> If you prefer to use an approach to relocating replicas and leases based on store IDs, see the [`ALTER RANGE ... RELOCATE`](alter-range-relocate.html) statement.
```
Comment (lunevalex) [inline v22.1/experimental-features.md:59]: replace store IDs with range IDs the benefit is really you get to use range ids instead of keys.

**Review by lunevalex — APPROVED** [review]

File: v22.1/alter-range-relocate.md (hunk 4 above)
Comment (rmloveland) [reply v22.1/alter-range-relocate.md:62]: Done!

File: v22.1/experimental-features.md (hunk 5 above)
Comment (rmloveland) [reply v22.1/experimental-features.md:59]: ok - updated, thanks!

**Comment by rmloveland** [comment]
Hi @ianjevans! mind giving this one a look?

**Review by ianjevans — APPROVED** [review]

**Comment by rmloveland** [comment]
Thanks Ian!  No worries, I drop stuff like ALL the time

**Pull request merged**